.github/workflows/automation-benchmark-tests.yml

```diff
@@ -43,6 +43,10 @@ on:
         description: Private funding key (Skip for Simulated)
         required: false
         type: string
+
+env:
+  BRANCH_NAME: ${{ github.head_ref || github.ref_name }}
```

Comment:
Is it possible that these values will be something other than a branch name? For example, would this be triggered by a tag or another ref? If so, amybe we call this just `REF_NAME` instead?

Reply:
Good call out, will fix.